app.go

```diff
@@ -1,5 +1,30 @@
 package control
 
+// A struct representing the settable fields of an Ably application.
+type NewApp struct {
+	//The application ID.
+	ID string `json:"id,omitempty"`
+	// The application name.
+	Name string `json:"name,omitempty"`
+	// The application status. Disabled applications will not accept
+	// new connections and will return an error to all clients.
+	Status string `json:"status,omitempty"`
+	// Enforce TLS for all connections. This setting overrides any channel setting.
+	TLSOnly bool `json:"tlsOnly"`
+	// The Firebase Cloud Messaging key.
+	FcmKey string `json:"fcmKey"`
+	// The Apple Push Notification service certificate.
+	// This field can only be used to set a new value,
+	// it will not be populated by queries.
+	ApnsCertificate string `json:"apnsCertificate"`
+	// The Apple Push Notification service private key.
+	// This field can only be used to set a new value,
+	// it will not be populated by queries.
+	ApnsPrivateKey string `json:"apnsPrivateKey"`
+	// The Apple Push Notification service sandbox endpoint.
```

Comment:
```suggestion
	// Use the Apple Push Notification service sandbox endpoint.
```

Reply:
Makes sense but these comments come from the control API docs word for word so I'd rather keep them in sync. Fixes can be pushed to them then pulled downstream.

Reply:
Sure, np, i've suggested it to the docs team